palm/plugins/dbt/commands/cmd_cleanup.py

```diff
@@ -9,7 +9,7 @@
 def cli(ctx):
     """Removes any artifacts from Snowflake related to the current branch."""
 
-    cmd = "dbt clean && dbt deps && dbt run-operation drop_branch_schemas"
+    cmd = "dbt run-operation drop_branch_schemas"
```

Comment:
why remove clean from cleanup? if so, then people have to shell in to run dbt clean when they need to

Reply:
My mistake, will fix!